tee-worker/service/src/prometheus_metrics.rs

```diff
@@ -244,6 +257,74 @@ fn handle_stf_call_request(req: RequestType, time: f64) {
 	observe_execution_time(category, label, time)
 }
 
+// This function will handle the conversion of the caller into a formatted string.
+fn format_caller(caller_option: Option<Identity>) -> String {
+	if let Some(caller) = caller_option {
+		// Note: This unwrap should be unfallable technically
+		return caller.to_account_id().unwrap().to_string()
+	}
+	"0xDeadBeef".to_string()
+}
+
+// This function will increment the metric with provided label values.
+fn inc_succesfull_trusted_operation_metric(operation: &str, block_str: &str, caller: &str) {
```

Comment:
If it's going to be only incremented I think `counter` would fit better.

Reply:
It's still a gauge

```
	static ref ENCLAVE_SUCCESFUL_TRUSTED_OPERATION: IntGaugeVec =
		register_int_gauge_vec!("litentry_worker_enclave_succesful_trusted_operation", "Litentry Succesful Trusted Operation", &["call", "block", "caller"])
			.unwrap();
```

Reply:
Yea this value can only increase - so counter should be better

Reply:
My bad, Updated it